File: quan.cs/Test/Config/Item/Reward.cs
using Newtonsoft.Json.Linq;
using Quan.Config;
using Quan.Utils;

namespace Test.Config.Item
{
    /// <summary>
	/// 奖励<br/>
	/// 代码自动生成，请勿手动修改
	/// </summary>
    public class Reward : Bean
    {
        public readonly int itemId;

        public ItemConfig itemId_Ref => ItemConfig.Get(itemId);

        public readonly int itemNum;


        public Reward(JObject json) : base(json)
        {
            itemId = json["itemId"]?.Value<int>() ?? default;
            itemNum = json["itemNum"]?.Value<int>() ?? default;
        }

        public static Reward Create(JObject json)
        {
            return new Reward(json);
        }

        public override string ToString()
        {
            return "Reward{" +
                   "itemId=" + itemId.ToString2() +
                   ",itemNum=" + itemNum.ToString2() +
                   '}';
        }
    }
}

File: quan.java/quan-config/src/test/java/quan/config/common/Reward.java
package quan.config.common;

import com.alibaba.fastjson.*;
import quan.config.*;
import quan.config.loader.ConfigLoader;
import quan.config.item.ItemConfig;

/**
 * 奖励<br/>
 * 代码自动生成，请勿手动修改
 */
public class Reward extends Bean {

    public final int itemId;

    public final int itemNum;


    public Reward(JSONObject json) {
        super(json);

        this.itemId = json.getIntValue("itemId");
        this.itemNum = json.getIntValue("itemNum");
    }

    public static Reward create(JSONObject json) {
        return new Reward(json);
    }

    @Override
    public String toString() {
        return "Reward{" +
                "itemId=" + itemId +
                ",itemNum=" + itemNum +
                '}';

    }

}
